content/sensu-go/5.11/sensuctl/reference.md

```diff
@@ -329,6 +330,23 @@ To create the `pagerduty` handler in the current session namespace:
 sensuctl create --file pagerduty.yml
 {{< /highlight >}}
 
+## Deleting resources
+
+The `sensuctl delete` command allows you to delete resources by reading from STDIN or a flag configured file (`-f`).
+The `delete` command accepts Sensu resource definitions in `wrapped-json` and `yaml` and uses the same [resources types][3] as `sensuctl create`.
```

Comment:
Does `sensuctl delete` require 100% match between all resource attributes or just name and namespace?

Reply:
Just the name and namespace. However, the resource must be valid.